include/ceed/ceed.h

```diff
@@ -416,6 +416,7 @@ CEED_EXTERN int  CeedOperatorIsAtPoints(CeedOperator op, bool *is_at_points);
 CEED_EXTERN int  CeedCompositeOperatorAddSub(CeedOperator composite_op, CeedOperator sub_op);
 CEED_EXTERN int  CeedCompositeOperatorGetNumSub(CeedOperator op, CeedInt *num_suboperators);
 CEED_EXTERN int  CeedCompositeOperatorGetSubList(CeedOperator op, CeedOperator **sub_operators);
+CEED_EXTERN int  CeedCompositeOperatorGetSubByName(CeedOperator op, const char *op_name, CeedOperator *sub_op);
```

Comment:
I have no idea why, but clang-format is adamant that these blocks of function signatures should have 2 spaces before the function name instead of 1. Not true for the block above it, just this one and a few below. Weird.